docs/OCP_CI_Tutorials/Containers/Container_Creation_Guide.md

```diff
@@ -36,12 +40,15 @@ Figure 1 above shows a simple flow chart to demonstrate that there are really tw
 The containers created for a specific scenario should be able to run any interop test written for that scenario. When writing the tests for a CSPI scenario, try to think about how they will be executed inside of a container as well as how you will target a separate cluster.
 
 ### Creating Containerized Tests
+
 For the following example, a very simple Python test will be utilized. This is just to help wrap your head around the idea of testing within a container.
```

Comment:
```suggestion
The following example uses a very simple Python test. This examples is to help understand the basic concept of testing within a container.
```
```suggestion
For the following example, a very simple Python test will be utilized. This is just to help wrap your head around the idea of testing within a container.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)